drivers/spi/spi_mcux_dspi.c

```diff
@@ -14,20 +14,33 @@
 #define SYS_LOG_LEVEL CONFIG_SYS_LOG_SPI_LEVEL
 #include <logging/sys_log.h>
 
+#ifndef CONFIG_SPI_LEGACY_API
```

Comment:
You can remove the support for the legacy API.

Reply:
Not until after the mcr20a driver gets converted to the new api

Reply:
true, forgot that one